tests/integration/it_owner_privileges_test.py

```diff
@@ -142,3 +142,101 @@ def test_can_not_manage_locations_from_different_company(clean_app, db_session):
     assert not has_privilege(
         method=Method.DELETE, resource="location", id=location.id
     )
+
+
+def test_can_manage_users_from_same_company(clean_app, db_session):
+    my_company = Company(
+        name="Mothers Of Invention Inc.", code="code1", address="addr"
+    )
+    db_session.add(my_company)
+    db_session.commit()
+    boss = Employee(
```

Comment:
@paulodamaso no need to do this, you can just do smth like this:
```
from tests import factories


boss = factories.EmployeeFactory(company=my_company) 
```

Reply:
@vladarefiev I did, but it did not worked, so I'm reverting to previous logic

Reply:
@paulodamaso Ah, what is problem you have faced?

Reply:
@vladarefiev Something about trying to create the same employee twice, I didn't want to debug it since it would make me to spend more than my budget to this task, so I've left the way I did before, at least it's correct

Reply:
@paulodamaso 
As I can see it works at all.
https://github.com/timelesslounge/timelessis/blob/master/tests/integration/it_test_authorization.py

I'd like to see EmployeeFactory here, but it's out of my hands, let @emilianodellacasa decide.